source/mongodb-handshake/handshake.rst

```diff
@@ -167,10 +167,15 @@ provided as a BSON object. This object has the following structure::
             platform: "<string>",
             /* OPTIONAL */
             env: {
-                name: "<string>",         /* REQUIRED */
+                name: "<string>",         /* OPTIONAL */
```

Comment:
I'm checking to see if `env.name` is expected to be reported, as changing this field to optional may mess up internal processes relying on this metadata.

Reply:
Update: this change is fine, no problem with making `env.name` optional.